internal/service/cluster/resource_cluster.go

```diff
@@ -803,15 +810,21 @@ func resourceMongoDBAtlasClusterUpdate(ctx context.Context, d *schema.ResourceDa
 		}
 	}
 
+	replicationSpecs, err := expandReplicationSpecs(d)
```

Comment:
any specific reason for this check being outside of the if `d.HasChange("replication_specs")` ?

Reply:
right